exporter/awsemfexporter/getJsonString

```diff
@@ -0,0 +1,60 @@
+import{
```

Comment:
This file is not a `golang` file.

Reply:
Oh wait that shouldn't be there, I'll remove it once I'm back at my work device